site/zh-cn/tutorials/generative/dcgan.ipynb

```diff
@@ -0,0 +1,931 @@
+      "metadata": {
+        "colab_type": "text",
+        "id": "rF2x3qooyBTI"
+      },
+      "source": [
+        "# 深度卷积生成对抗网络"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "0TD5ZrvEMbhZ"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://tensorflow.google.cn/tutorials/generative/dcgan\">\n",
+        "    <img src=\"https://tensorflow.google.cn/images/tf_logo_32px.png\" />\n",
+        "    在 tensorFlow.google.cn 上查看</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/zh-cn/tutorials/generative/dcgan.ipynb\">\n",
+        "    <img src=\"https://tensorflow.google.cn/images/colab_logo_32px.png\" />\n",
+        "    在 Google Colab 中运行</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/zh-cn/tutorials/generative/dcgan.ipynb\">\n",
+        "    <img src=\"https://tensorflow.google.cn/images/GitHub-Mark-32px.png\" />\n",
+        "    在 GitHub 上查看源代码</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/zh-cn/tutorials/generative/dcgan.ipynb\"><img src=\"https://tensorflow.google.cn/images/download_logo_32px.png\" />下载 notebook</a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "M0gHG-LEgLZx",
+        "colab_type": "text"
+      },
+      "source": [
+        "Note: 我们的 TensorFlow 社区翻译了这些文档。因为社区翻译是尽力而为， 所以无法保证它们是最准确的，并且反映了最新的\n",
+        "[官方英文文档](https://www.tensorflow.org/?hl=en)。如果您有改进此翻译的建议， 请提交 pull request 到\n",
+        "[tensorflow/docs](https://github.com/tensorflow/docs) GitHub 仓库。要志愿地撰写或者审核译文，请加入\n",
+        "[docs-zh-cn@tensorflow.org Google Group](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-zh-cn)。"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "ITZuApL56Mny"
+      },
+      "source": [
+        "本教程演示了如何使用[深度卷积生成对抗网络](https://arxiv.org/pdf/1511.06434.pdf)（DCGAN）生成手写数字图片。该代码是使用 [Keras Sequential API](https://tensorflow.google.cn/guide/keras) 与 `tf.GradientTape` 训练循环编写的。"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "2MbKJY38Puy9"
+      },
+      "source": [
+        "## 什么是生成对抗网络？\n",
+        "\n",
+        "[生成对抗网络](https://arxiv.org/abs/1406.2661)（GANs）是当今计算机科学领域最有趣的想法之一。两个模型通过对抗过程同时训练。一个*生成器*（“艺术家”）学习创造看起来真实的图像，而*判别器*（“艺术评论家”）学习区分真假图像。\n",
+        "\n",
+        "![生成器和判别器图示](https://github.com/Berrywrq/docs/blob/master/site/zh-cn/tutorials/generative/images/gan1.png?raw=1)\n",
```

Comment:
Please don't link to your personal github repo.

Perhaps link to the en/ version: https://github.com/tensorflow/docs/blob/master/site/en/tutorials/generative/images/gan1.png